test/mysql_change_user_tests.erl

```diff
@@ -136,20 +137,17 @@ execute_queries_test() ->
 execute_queries_failure_test() ->
     Pid = connect_db(?user1, ?password1),
     erlang:process_flag(trap_exit, true),
-    {ok, Ret, Logged} = error_logger_acc:capture(
-        fun () ->
-            mysql:change_user(Pid, ?user2, ?password2, [{queries, [<<"foo">>]}])
-        end),
+    {ok, Ret, Logged} = error_logger_acc:capture(fun () ->
+        Ret1 = mysql:change_user(Pid, ?user2, ?password2, [{queries, [<<"foo">>]}]),
```

Comment:
I don't mind this style and I'm not asking you to change it. Just want to mention that I believe indenting `end` directly under `fun`, `case`, etc. is more typical in Erlang code, e.g.

```Erlang
    {ok, Ret, Logged} = error_logger_acc:capture(
        fun () ->
            ...
        end),

Reply:
Hm, actually I prefer the style you just outlined. The one I used there, I mimicked from another test...